components/ui/doorhanger/src/main/java/mozilla/components/ui/doorhanger/DoorhangerPrompt.kt

```diff
@@ -129,6 +140,9 @@ class DoorhangerPrompt(
                 return (view as android.widget.RadioButton).apply {
                     this.id = viewId
                     this.text = label
+                    this.setOnClickListener {
+                        onClick?.invoke()
+                    }
```

Comment:
This shouldn't be needed since the group already updates the checked state. You can get the checked item from the control group.

Reply:
👍